src/Views/AppListView.vala

```diff
@@ -48,6 +50,21 @@ namespace AppCenter.Views {
 
             add (scrolled);
         }
+        
+        public void set_flathub_uri (string uri) {
+            /*
+                Atheesh Thirumalairajan (@candiedoperation)
+                https://github.com/candiedoperation
+                
+                This Method has been added in order to provide
+                dynamic flathub.org links for current searched
+                text in the App Search Box
+            */
```

Comment:
Same comment as before. If you want to add yourself to copyright, please do so in the copyright header. Otherwise, there's git blame for this sort of thing.